src/validation.cpp

```diff
@@ -2771,14 +2771,6 @@ void CChainState::UpdateTip(const CBlockIndex* pindexNew)
                 }
             }
         }
-        // Check the version of the last 100 blocks to see if we need to upgrade:
-        for (int i = 0; i < 100 && pindex != nullptr; i++)
-        {
-            int32_t nExpectedVersion = ComputeBlockVersion(pindex->pprev, m_params.GetConsensus());
-            if (pindex->nVersion > VERSIONBITS_LAST_OLD_BLOCK_VERSION && (pindex->nVersion & ~nExpectedVersion) != 0)
-                ++nUpgraded;
-            pindex = pindex->pprev;
-        }
```

Comment:
20054: looks incomplete/missing some previous backport

Reply:
removed 20054 for now , will check for dependency and then backport it